packages/fluentui/accessibility/src/behaviors/Breadcrumb/breadcrumbItemBehavior.ts

```diff
@@ -0,0 +1,15 @@
+import { Accessibility } from '../../types';
+
+/**
+ * @specification
+ * Adds role='listitem' if element type is 'breadcrumbitem'. This allows screen readers to handle the component as a listitem.
```

Comment:
`@specification` tag is used for automatic unit tests based on JSDoc

```suggestion
 * Adds role='listitem'.
```

https://github.com/microsoft/fluentui/blob/master/packages/fluentui/test-a-feature.md#behavior-tests